packages/forma-36-website/src/pages/components/icon/index.mdx

```diff
@@ -0,0 +1,207 @@
+---
+title: 'Icon'
+type: 'component'
+github: 'https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components/src/components/Icon'
+storybook: 'https://f36-storybook.contentful.com/?path=/story/components-icon--icon-default'
+---
+
+(Icons are visual hints for users to indicate actions, statuses and so on. Is used in Buttons, Icon Buttons and so on.)
+
+```jsx
+<React.Fragment>
+  <Icon icon="Edit" />
+</React.Fragment>
+```
+
+## Table of contents
+
+1. [Best practices](#best-practices)
+2. [Variations](#variations)
+   1. [Colors](#colors)
+   2. [Size](#size)
+   3. [Icon type](#icon)
+3. [Copy considerations](#copy-considerations)
+4. [Accessibility](#accessibility)
+
+## Best practices <a name="best-practices" />
+
+- Position buttons consistently in the interface
+- Use right Icons to represent right actions
+- Use extablished Icons
+- Pair Icons with text
+
+## Variations <a name="variations" />
+
+Icon component can be configurated in different ways. Here are some examples.
```

Comment:
The `Icon` component can be configured in different ways using variations in color, size and icon type: